File: consecutive-characters/consecutive-characters.cpp
class Solution {
public:
    int maxPower(string s) {
        int cnt = 0 ;
        int maxi = 0 ;
        for(int i = 0 ; i < s.size()-1 ; i++){
            if(s[i]==s[i+1]){
                cnt++;
                if(cnt>maxi){
                    maxi = cnt ;
                }
            }
                else{
                    cnt = 0 ;
                }
            
        }
        return maxi+1;
    }
};

File: consecutive-characters/consecutive-characters.py
class Solution:
    def maxPower(self, s: str) -> int:
        cnt = 0
        maxi = 0
        for i in range(len(s)-1):
            if s[i]==s[i+1]:
                cnt += 1
                if cnt > maxi:
                    maxi = cnt 
            else:
                cnt = 0
                
        return maxi+1
            